content/blog/2021-09-22-refactor.md

```diff
@@ -0,0 +1,163 @@
+﻿---
+title: Easy Stuctural Refactors to Python Source Code
+date: 2021-09-22
+description: |
+  Simple, hassle-free, dependency-free, AST based source code refactoring toolkit.
+
+descriptionLong: |
+  [Refactor](https://github.com/isidentical/refactor) is a source code 
+  refactoring engine that takes the advantage of the AST module in the standard
+  library. Through using it, we can find-and-fix patterns in massive codebases.
+
+picture: 2021-09-22/ast.png
+author: batuhan_taskaya
+commentsUrl: https://discuss.dvc.org/t/easy-structural-refactors-to-python-source-code/895
+tags:
+  - Refactor
```

Comment:
can we please introduce `engineering` and `open source` (or `open-source`?) tags? Probably propagate them to all things that were written by the eng team - Peter, Casper

Reply:
in this PR or in a follow-up?

Reply:
Adding tags to this post.  Let's add the tags to the others separately pls.